### ISSUE TITLE
Allow edit pages to specific usegroups (moderators, etc)

### ISSUE THREAD

**Issue by Cu8eeeR** [Sama34/OUGC-Pages#10]

**Comment by Sama34** [comment]
So you want to be able to edit the page content from the front end? I think we could add a option for this in the ACP and make use of the MyCode editor there.

**Comment by Sick1G** [comment]
Yes, this is awesome. We could make actually a Wiki-like on Reddit which is updated by the community members which have permission to edit wiki page

This is an example: https://www.reddit.com/r/CreativeCSS/wiki/index

**Comment by Sama34** [comment]
The issue is that this plugin allows for PHP pages, which to be cautious about. Also, what kind of restrictions or data recovery options would you consider if any?

**Comment by Eldenroot** [comment]
Yes, that could be an issue... however we can add a permission system - just usergroup select who can edit the page and who can edit the page including other options

**Comment by Sama34** [comment]
What would be the difference to using xThreads if I may ask?

**Comment by Sama34** [comment]
It could also be possible to ship this as an example page. This would have the benefit of using already existing features as to not duplicate everything on the front-end. The following settings could be included inside the page contents :
- Edit only pages from a single category (to allow only editing of non-PHP pages, for example)
- Limit the fields allowed to edit, for instance: _Viewable for Groups_, _Add to Menu_, _Use Template Wrapper_, _Show In Who Is Online (WOL) List_, _Page Content_

Of course any and all settings from the parent category and for the page itself would be inhered to restrict usage of the page.

I'm also thinking about shipping complex example pages that ship with [Hooks ](https://github.com/frostschutz/MyBB-Hooks) import files. This could be useful in this specific case for instance to add a link to the ModCP or to handle addition of templates and thus not overhaul the page itself with HTML code.

If you want to contribute by coding the page we could see into shipping it without necessarily waiting for a full plugin release. I plan to setup a way for users to get individual pages from.
